### PR TITLE
[RUN-3892] Add section about Java version selection (Studio Pro 10.8.0)

### PR DESCRIPTION
**Pull request by atennapel**

### CODE REVIEW

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 1]
```diff
@@ -67,7 +67,13 @@ The URL prefix must be alphanumeric. It cannot be empty, contain whitespace, or
 
 If the URL prefix breaks any of the rules mentioned above, then you will get a consistency error.
 
-### 3.5 After Startup{#after-startup}
+### 3.5 Java version{#java-version}
+
+Here you can select which Java version to use for you application.
+
+For local development the JDK configured in the Studio preferences has to be compatible with the Java version configured here.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: Studio _Pro_ preferences?
Where is this set in the Studio Pro preferences?

Comment (atennapel) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md]: I added a link to the page, could you check if it is correct?

File: content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md [hunk 2]
```diff
@@ -67,7 +67,13 @@ The URL prefix must be alphanumeric. It cannot be empty, contain whitespace, or
 
 If the URL prefix breaks any of the rules mentioned above, then you will get a consistency error.
 
-### 3.5 After Startup{#after-startup}
+### 3.5 Java version{#java-version}
+
+Here you can select which Java version to use for you application.
+
+For local development the JDK configured in the [Studio Pro preferences](/refguide/preferences-dialog#jdk-directory) has to be compatible with the Java version configured here.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md:74]: @atennapel 
Hi Albert - link was slightly wrong. In the refguide the pages don't quite follow the indenting (for historical reasons - we changed the way the navigation works).

Does this make a difference for apps deployed to the cloud? Will they use this value and automatically deploy using the correct version?

Comment (atennapel) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md:74]: > Does this make a difference for apps deployed to the cloud? Will they use this value and automatically deploy using the correct version?

Yes that is correct. Would you suggest to add a sentence about that?
"Applications deployed to the cloud will use this setting to select which Java version to use."

Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md:74]: I think that would be useful. If I had the questions, then perhaps other people might be in doubt? Can you do that?

Comment (atennapel) [reply content/en/docs/refguide/modeling/app-explorer/app/app-settings/_index.md:74]: Done.

**Review by MarkvanMents — APPROVED** [review]
Thanks, Albert.
All looks good. Approving for Mendix 10.8

**Pull request merged**